### DISCUSSION TITLE
Github Education and Classroom

### DISCUSSION THREAD

**Discussion by AmitChigare** [DiveHQ/backend-internship-task#32] [Q&A]
Hello Sir, I have recently created github education account, it has already got approved but takes time to give all the developer features. I cannot access the Github Classroom page too, it is saying all are full. I have created a new pull request with my project. It is sufficient? Please consider the issue and help me out to add the submission.

**Comment by prayashm** [comment]
Right now, we aren't accepting submissions outside of Github classroom
